jobsdb/unionQueryLegacy.go

```diff
@@ -1,53 +1,84 @@
 package jobsdb
 
-import "context"
+import (
+	"context"
+	"fmt"
+)
 
 type MultiTenantLegacy struct {
 	*HandleT
 }
 
-func (mj *MultiTenantLegacy) GetAllJobs(ctx context.Context, workspaceCount map[string]int, params GetQueryParamsT, _ int) ([]*JobT, error) { // skipcq: CRT-P0003
+type legacyMoreToken struct {
```

Comment:
Explain: for legacy query we need to keep track of each subQuery's maximum job ID